### PR TITLE
add detail this also logs event 4625

### PR DESCRIPTION
**Pull request by HerbertMauerer**
We may have customers seeing event 4625 as additional hint the task failed.

### CODE REVIEW

**Comment by HerbertMauerer** [comment]
oh, two typos, sorry about it. Better:
failure event 4625 with error 0xC0000064

File: windows/security/identity-protection/credential-guard/credential-guard-known-issues.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The following known issue has been fixed in the [Cumulative Security Update for
    Failure occurred in ‘LogonUserExEx’. <br>
    User Action: Ensure the credentials for the task are correctly specified. <br>
    Additional Data: Error Value: 2147943726. 2147943726: ERROR\_LOGON\_FAILURE (The user name or password is incorrect)."
--  When enabling NTLM audit on the domain controller, an Event ID 8004 with an indecipherable username format is logged. For example:
+-  When enabling NTLM audit on the domain controller, an Event ID 8004 with an indecipherable username format is logged. You get a similar user name in a user logon failure event 4525 wiith error 0xC0000064 on the machine itself. For example:
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/credential-guard/credential-guard-known-issues.md:36]:
```suggestion
-  When enabling NTLM audit on the domain controller, an Event ID 8004 with an indecipherable username format is logged. You also get a similar user name in a user logon failure event 4525 with error 0xC0000064 on the machine itself. For example:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @HerbertMauerer, kindly commit the suggested changes. Many thanks.

**Review by denisebmsft — APPROVED** [review]
Thank you @HerbertMauerer @JohanFreelancer9

**Pull request merged**